### DISCUSSION TITLE
How to erase 'saved articles' without losing other things?

### DISCUSSION THREAD

**Discussion by serrq** [spacecowboy/Feeder#360] [Ideas]
I have about 2000 saved articles.

I want wipe out all.

**Comment by spacecowboy** [comment]
No easy way at the moment I'm afraid

**Comment by emigrant-song** [comment]
Hello, I’d share a possible workaround for erasing saved articles.

1. Export feeds to OPML.
2. Delete all feeds by "Delete feed".
3. Import the previously exported OPML.

Note:
- Keep the amount of traffic in mind. When importing, a certain amount of articles will be displayed as unread again. It is recommended that you mark all articles as read before doing this.
- Check the exported OPML for any problems before deleting feeds.

**Comment by serrq** [comment]
Modular backup is another solution.
One can backups just what it wants. And it can reinstall the tailored backup afterwards.

Of course each restore overrides the current database.

![1000045980](https://github.com/user-attachments/assets/549a4cec-1ad9-49dc-bad0-479196e802b3)
